madmom/audio/spectrogram.py

```diff
@@ -611,18 +613,31 @@ def __new__(cls, spectrogram, log=LOG, mul=MUL, add=ADD, **kwargs):
         obj.add = add
         # and those from the given spectrogram
         obj.stft = spectrogram.stft
-        obj.bin_frequencies = spectrogram.bin_frequencies
-        obj.filterbank = spectrogram.filterbank
+        obj.spectrogram = spectrogram
         # return the object
         return obj
 
     def __array_finalize__(self, obj):
         if obj is None:
             return
         # set default values here, also needed for views
+        self.stft = getattr(obj, 'stft', None)
+        self.spectrogram = getattr(obj, 'spectrogram', None)
         self.mul = getattr(obj, 'mul', MUL)
         self.add = getattr(obj, 'add', ADD)
-        super(LogarithmicSpectrogram, self).__array_finalize__(obj)
+
+    @property
+    def filterbank(self):
+        """Filterbank."""
+        return self.spectrogram.filterbank
+
+    @property
+    def bin_frequencies(self):
+        """Bin frequencies."""
+        try:
```

Comment:
Is this necessary? As far as I understand, the `self.spectrogram` is a `FilteredSpectrogram` (and thus has a filterbank), then `self.spectrogram.bin_frequencies` will return the `center_frequencies` of the filterbank anyways.